builds/e2e/e2e.yaml

```diff
@@ -35,15 +35,44 @@ jobs:
     timeoutInMinutes: 120
 
     steps:
-    - bash: |
-        # We use a self-hosted agent for this job, so we need to handle
-        # clean up root-owned files and docker images ourselves (anything
-        # created during `dotnet vstest ...`)
-        sudo git clean -ffdx || true
-        sudo docker rm -f $(docker ps -a -q) || true
-        sudo docker system prune -a -f --volumes || true
-      displayName: Clean up files and images
+    - pwsh: |
+        sudo git clean -ffdx
```

Comment:
how about docker network? do we need to explicitly remove it as well.  We sometimes got into a state that docker network corrupted.

Reply:
Yep, I'm removing the docker network a little farther down (in the other `pwsh` task). https://github.com/Azure/iotedge/blob/8562f6ce70c3b927b3b9546fd432a53457305bf7/builds/e2e/e2e.yaml#L70